content/scout/integrations/environment/sysdig.md

```diff
@@ -0,0 +1,96 @@
+images automatically, enable a [registry integration](../_index.md#container-registries).
+
+Image analysis must not necessarily precede the runtime integration, but the
+environment assignment only takes place once Docker Scout has analyzed the
+image.
+
+## Prerequisites
+
+- Install the Sysdig Agent in the cluster that you want to integrate, see [Install Sysdig Agent (Sysdig docs)](https://docs.sysdig.com/en/docs/installation/sysdig-monitor/install-sysdig-agent/).
+- Enable profiling for Risk Spotlight Integrations in Sysdig, see [Profiling (Sysdig docs)](https://docs.sysdig.com/en/docs/sysdig-secure/policies/image-profiles/#enable-for-risk-spotlight-integrations-or-for-the-in-use-column).
+- You must be an organization owner to enable the integration in the Docker Scout Dashboard.
+
+## Integrate an environment
+
+1. Go to [Integrations](https://scout.docker.com/settings/integrations/) on the
+   Docker Scout Dashboard.
+2. Select the **Detect images running in my environments** filter.
+3. Find Sysdig in the list, and select **Integrate**.
+
+4. In the **How to integrate** section, enter a configuration name for this
+   integration. Docker Scout uses this label as a display name for the
+   integration.
+
+5. Select **Next**.
+
+6. Enter a Risk Spotlight API token and select the region in the drop-down
+   list.
+
+   The Risk Spotlight API token is the Sysdig token that Docker Scout needs to
+   integrate with Sysdig. For more instructions on how to generate a Risk
+   Spotlight token, See [Risk Spotlight Integrations (Sysdig docs)](https://docs.sysdig.com/en/docs/sysdig-secure/integrations-for-sysdig-secure/risk-spotlight-integrations/#generate-a-token-for-the-integration).
+
+   The region corresponds to the `global.sysdig.region` configuration parameter
+   set when deploying the Sysdig Agent.
+
+7. Select **Next**.
+
+   After selecting **Next**, Docker Scout connects to Sysdig and retrieves the
+   cluster names for your Sysdig account. Cluster names correspond to the
+   `global.clusterConfig.name` configuration parameter set when deploying
+   Sysdig Agents.
+
+   An error displays if Docker Scout fails to connect to Sysdig using the
+   provided token. If there's an error, you won't be able to continue the
+   integration. Go back and verify that the configuration details are correct.
+
+8. Select a cluster name in the drop-down list.
+
+9. Select **Next**.
+
+10. Assign an environment name for this cluster.
+
+    You can reuse an existing environment or create a new one.
+
+11. Select **Enable integration**.
+
+After enabling the integration, Docker Scout automatically detects images
+running in the cluster, and assigns those images to the environment associated
+with the cluster.
+
+To integrate more clusters, go to the [Integrations](https://scout.docker.com/settings/integrations/ecr),
+page, select **Sysdig** > **Manage**, and select the **Add** button.
```

Comment:
**suggestion (non-blocking):** we're working on making it clearer for customers that if they're using a new environment it won't even show up until they push a new image into Sysdig. I think the docs could be a wonderful place to get that point across as well 🙏

Reply:
Hmmm. Could I ask for a clarification?

Does this mean that an image must be redeployed to the cluster if the environment is new? If the environment already exists and holds one or more image references, a redeploy is not necessary?

Or does it mean that the image analysis results must be pushed to the platform for the environment to appear?

Reply:
In essence, it's the very last one you mention. For the user it means, when they enable the Sysdig integration with a new environment:
- There must be a Scout enabled repo running on that cluster
- Or they push an image from a Scout enabled repo into that cluster

Only with these requirements, **and after Scout has the chance to detect those images on Sysdig**, will the user see the new env on the dropdown. It's not instant, when I tried on my own it was taking around 30s to a minute

Reply:
OK, thanks!